hdmi2usb/modeswitch/boards.py

```diff
@@ -145,12 +158,57 @@ def load_fx2(board, mode=None, filename=None, verbose=False):
             raise
 
 
+def load_fx2_dfu_bootloader(board, verbose=False, filename='boot-dfu.ihex',
+                            vid_pid=(0x04B4, 0x8613)):
+    """
+    Loads bootloader firmware onto given board and updates the board to point
+    to correct device. The device is identified using bootloader's VID and PID.
```

Comment:
Can you instead use the location in the USB tree? That should be guaranteed not to change?